sources/packages/api/src-sql/Applications/Add-col-relationship-status.sql

```diff
@@ -0,0 +1,13 @@
+-- Create relationship_status
+ALTER TABLE
+    sims.applications
+ADD
+    COLUMN IF NOT EXISTS relationship_status VARCHAR(20) NOT NULL DEFAULT 'single';
```

Comment:
how about creating enum for relationship statues? @devs thoughts?

Reply:
Agree with @ann-aot. It may be re-used for future as well.

Reply:
if that is the case, we should have values, as married, single and other. @abschwenker  please confirm

Reply:
since we are using this field is other areas and doing a translation between status and a "code" yes i would agree that ENUM would fit